### PR TITLE
fix: added max_completion_tokens for chat complete

### PR DESCRIPTION
**Pull request by shivam-pareek**
**Title:** 
- max_completion_tokens has been integrated

**Description:** (optional)
- To keep the providers open ai compliant

### CODE REVIEW

File: src/providers/google-vertex-ai/chatComplete.ts
Comment (VisargD) [inline src/providers/google-vertex-ai/chatComplete.ts:1]: For Google Vertex, you will also have to add the mapping in `transformGenerationConfig` function in `transformGenerationConfig.js` file

File: src/providers/google/chatComplete.ts
Comment (VisargD) [inline src/providers/google/chatComplete.ts:1]: Please also update the `transformGenerationConfig` function present in this same file

File: src/providers/predibase/chatComplete.ts [hunk 1]
```diff
@@ -35,6 +35,12 @@ export const PredibaseChatCompleteConfig: ProviderConfig = {
     default: 4096,
     min: 0,
   },
+  max_completion_tokens: {
+    param: 'max_completion_tokens',
```
Comment (VisargD) [inline src/providers/predibase/chatComplete.ts]: Please fix this mapping. It should be `max_tokens`

File: src/providers/azure-openai/chatComplete.ts [hunk 2]
```diff
@@ -27,6 +27,11 @@ export const AzureOpenAIChatCompleteConfig: ProviderConfig = {
     default: 100,
     min: 0,
   },
+  max_completion_tokens: {
+    param: 'max_tokens',
```
Comment (VisargD) [inline src/providers/azure-openai/chatComplete.ts]: For azure-openai, we can keep the mapping to `max_completion_tokens`

**Comment by VisargD** [comment]
@shivam-pareek - Bedrock has multiple chat configs for different sub-providers. Can you please add mapping to all of them? Same for Vertex.

**Comment by shivam-pareek** [comment]
> @shivam-pareek - Bedrock has multiple chat configs for different sub-providers. Can you please add mapping to all of them? Same for Vertex.

ya sure

**Review by VisargD — APPROVED** [review]

**Pull request merged**